### PR TITLE
Allow for tracking which kit ID an account was created using

### PR DESCRIPTION
**Pull request by wasade**
In brief: modify account table, recover kit IDs from old accounts, and modify the model/repo code to allow for getting a kit name in and out from the `accounts` table.

### CODE REVIEW

File: microsetta_private_api/api/implementation.py [hunk 1]
```diff
@@ -157,6 +158,7 @@ def update_account(account_id, body, token_info):
             body['address']['post_code'],
             body['address']['country_code']
         )
+        acc.created_with_kit_name = body.get('kit_name')
```
Comment (dhakim87) [inline microsetta_private_api/api/implementation.py:161]: This doesn't make sense to me.  If the purpose of this field is to track what was sent when the user created the kit, we shouldn't allow them to edit the field later when they update account information.  Unless you want this to be possible for admins for some reason (in which case it should validate admin permission)?

File: microsetta_private_api/repo/account_repo.py [hunk 2]
```diff
@@ -14,11 +14,11 @@ def __init__(self, transaction):
                 "account_type, auth_issuer, auth_sub, " \
                 "first_name, last_name, " \
                 "street, city, state, post_code, country_code, " \
-                "creation_time, update_time"
+                "creation_time, update_time, created_with_kit_id"
```
Comment (dhakim87) [inline microsetta_private_api/repo/account_repo.py:17]: The column ordering is a little bit inconsistent in this file.  In the reads it follows update time, in the writes it follows last name.  I'd prefer it immediately followed country_code (and preceded creation_time, update_time if applicable) in all reads and writes in the file.

File: microsetta_private_api/repo/account_repo.py [hunk 3]
```diff
@@ -14,11 +14,11 @@ def __init__(self, transaction):
...
 
     write_cols = "id, email, " \
                  "account_type, auth_issuer, auth_sub, " \
-                 "first_name, last_name, " \
+                 "first_name, last_name, created_with_kit_id, " \
```
Comment (dhakim87) [inline microsetta_private_api/repo/account_repo.py:21]: Ideally would follow country code

File: microsetta_private_api/repo/account_repo.py [hunk 4]
```diff
@@ -41,14 +41,15 @@ def _row_to_account(r):
...
+            r['created_with_kit_id'],
             r['creation_time'], r['update_time'])
 
     @staticmethod
     def _account_to_row(a):
         return (a.id, a.email,
                 a.account_type, a.auth_issuer, a.auth_sub,
-                a.first_name, a.last_name) + \
-               AccountRepo._addr_to_row(a.address)
+                a.first_name, a.last_name, a.created_with_kit_id) + \
```
Comment (dhakim87) [inline microsetta_private_api/repo/account_repo.py:51]: Ideally would follow country code

File: microsetta_private_api/repo/account_repo.py [hunk 5]
```diff
@@ -146,6 +146,7 @@ def update_account(self, account):
                             "auth_sub = %s, "
                             "first_name = %s, "
                             "last_name = %s, "
+                            "created_with_kit_id = %s, "
```
Comment (dhakim87) [inline microsetta_private_api/repo/account_repo.py:149]: If you decide to make my requested change in ordering of columns, this line has to be shifted.

File: microsetta_private_api/api/implementation.py (hunk 1 above)
Comment (wasade) [reply microsetta_private_api/api/implementation.py:161]: Makes sense, will remove from update

File: microsetta_private_api/repo/account_repo.py (hunk 2 above)
Comment (wasade) [reply microsetta_private_api/repo/account_repo.py:17]: Okay, ill take a look

**Pull request merged**